<launch>
    <node name="cone_detection" pkg="rr_iarrc" type="cone_detection" output="screen">

        <!--The larger the height the lower it will be on the screen-->
        <param name="blockSky_height" type="int" value="200" />
        <param name="blockWheels_height" type="int" value="200" />
        <param name="blockBumper_height" type="int" value="200" />

        <param name="subscription_node" type="string" value="/camera/image_color_rect"/>
    </node>

    <include file="$(find rr_platform)/launch/sedani_state_publisher.launch"/>
    <include file="$(find rr_iarrc)/launch/lines_pcl_converter.launch"/>

</launch>
